macros/EmbedTest262ReportResultsTable.ejs

```diff
@@ -0,0 +1,12 @@
+<%
+// Embeds a Test262.report results table
+//
+// Parameters:
+//  $0 - A "feature" name from test262.report
+//
+//  Example calls:
+// {{EmbedTest262ReportResultsTable("globalThis")}}
+// {{EmbedTest262ReportResultsTable("BigInt")}}
+//
+%>
+<iframe frameborder="0" scrolling="no" style="width: 100%; height: 300px;" src="https://test262.report/embed/features/<%-encodeURIComponent($0)%>?engines=chakra%2Cjavascriptcore%2Cspidermonkey%2Cv8&summary=true&include-browsers=true"></iframe>
```

Comment:
```suggestion
<iframe width="100%" height="300px" src="https://test262.report/embed/features/<%-encodeURIComponent($0)%>?engines=chakra%2Cjavascriptcore%2Cspidermonkey%2Cv8&summary=true&include-browsers=true"></iframe>
```

Reply:
I’d recommend adding line breaks like in https://github.com/mdn/kumascript/pull/1245#discussion_r330833257, which is what Prettier gave me.

Reply:
@ExE-Boss is this the new preferred convention? I don't see any other examples like this in the existing macros 🤷‍♀

Reply:
No, it's not the preferred convention, since we don't use `prettier` for `ejs` files. The single-line approach you have is fine.